readthedocs/templates/core/project_bar_base.html

```diff
@@ -41,7 +43,7 @@
     {% if project.skip %}
       <p class="build-failure">
             {% blocktrans %}
-            Your project is currently disabled for abuse of the system. 
+            Your project is currently disabled for abuse of the system.
```

Comment:
My text editor did this automatically. Do you want me to undo it or do it as a separate commit ? From my point of view, it's better this way.